docs/intro/intro_scaling.rst

```diff
@@ -104,8 +107,8 @@ Time the construction:
     Because runtimes are highly system dependent, running the code on your system
     will likely result in different values. The results shown here are illustrative.
 
-Summation
-=========
+Use ``quicksum``
```

Comment:
I think the code formatting in titles looks strange. IMO we can just say `Use quicksum` without the formatting.